Marvel/Characters/Spider-Man/CBRO/Miles Morales (CBRO) with events.cbl

```diff
@@ -0,0 +1,1565 @@
+<Database Name="cv" Series="85928" Issue="548571" />
+</Book>
+<Book Series="All-New, All-Different Avengers" Number="15" Volume="2015" Year="2016">
+<Database Name="cv" Series="85928" Issue="555510" />
+</Book>
+<Book Series="Captain Marvel" Number="8" Volume="2016" Year="2016">
+<Database Name="cv" Series="87428" Issue="546052" />
+</Book>
+<Book Series="Ms. Marvel" Number="11" Volume="2015" Year="2016">
+<Database Name="cv" Series="86114" Issue="551310" />
+</Book>
+<Book Series="Power Man and Iron Fist" Number="7" Volume="2016" Year="2016">
+<Database Name="cv" Series="88214" Issue="544991" />
+</Book>
+<Book Series="Power Man and Iron Fist" Number="8" Volume="2016" Year="2016">
+<Database Name="cv" Series="88214" Issue="550367" />
+</Book>
+<Book Series="Power Man and Iron Fist" Number="9" Volume="2016" Year="2016">
+<Database Name="cv" Series="88214" Issue="553018" />
+</Book>
+<Book Series="Mockingbird" Number="6" Volume="2016" Year="2016">
+<Database Name="cv" Series="88764" Issue="544989" />
+</Book>
+<Book Series="New Avengers" Number="13" Volume="2015" Year="2016">
+<Database Name="cv" Series="85301" Issue="539247" />
+</Book>
+<Book Series="New Avengers" Number="14" Volume="2015" Year="2016">
+<Database Name="cv" Series="85301" Issue="541215" />
+</Book>
+<Book Series="New Avengers" Number="15" Volume="2015" Year="2016">
+<Database Name="cv" Series="85301" Issue="546062" />
+</Book>
+<Book Series="New Avengers" Number="16" Volume="2015" Year="2016">
+<Database Name="cv" Series="85301" Issue="551311" />
+</Book>
+<Book Series="New Avengers" Number="17" Volume="2015" Year="2016">
+<Database Name="cv" Series="85301" Issue="555533" />
+</Book>
+<Book Series="New Avengers" Number="18" Volume="2015" Year="2017">
+<Database Name="cv" Series="85301" Issue="562601" />
+</Book>
+<Book Series="Ultimates" Number="9" Volume="2015" Year="2016">
+<Database Name="cv" Series="85972" Issue="540090" />
+</Book>
+<Book Series="Squadron Supreme" Number="9" Volume="2015" Year="2016">
+<Database Name="cv" Series="86716" Issue="540084" />
+</Book>
+<Book Series="Squadron Supreme" Number="10" Volume="2015" Year="2016">
+<Database Name="cv" Series="86716" Issue="542631" />
+</Book>
+<Book Series="Squadron Supreme" Number="11" Volume="2015" Year="2016">
+<Database Name="cv" Series="86716" Issue="548585" />
+</Book>
+<Book Series="Squadron Supreme" Number="12" Volume="2015" Year="2016">
+<Database Name="cv" Series="86716" Issue="552171" />
+</Book>
+<Book Series="Guardians of the Galaxy" Number="11" Volume="2015" Year="2016">
+<Database Name="cv" Series="85290" Issue="547277" />
+</Book>
+<Book Series="Rocket Raccoon &#38; Groot" Number="8" Volume="2015" Year="2016">
+<Database Name="cv" Series="86989" Issue="547289" />
+</Book>
+<Book Series="Rocket Raccoon &#38; Groot" Number="9" Volume="2015" Year="2016">
+<Database Name="cv" Series="86989" Issue="549522" />
+</Book>
+<Book Series="Rocket Raccoon &#38; Groot" Number="10" Volume="2015" Year="2016">

```

Comment:
Series year, so volume, is 2016 for this one, looks like
https://comicvine.gamespot.com/rocket-raccoon-and-groot/4050-86989/